xml/schema/libraryFiles.xml

```diff
@@ -49,6 +49,10 @@
 		<field name="submission_id" type="I8">
 			<NOTNULL/>
 		</field>
+		<field name="public_access" type="I1">
+				<NOTNULL/>
```

Comment:
Off the top of my head, ADODB doesn't always handle new columns with `NOT NULL` constraints, even though there's a `DEFAULT VALUE` here. I'd suggest removing the `NOT NULL`. It'll work just fine because `NULL` values will be the default, and they're equivalent to boolean `false`.